AA-427: Remove all mentions of 'aggregation' possible (#8)
eth-infinitism/ERCs: ERCS/erc-4337.md

### Required entry point contract functionality

> There is one main entry point method: `handleOps`.

Reviewer: ```suggestion
The entry point method is `handleOps`, which handles an array of userOps
```

> * `handleOps` handles userOps of accounts.

Reviewer: ```suggestion

```

In addition, we need to verify that the same will hold true when executed on-chain.
For this purpose, a UserOperation is not allowed to access any information that might change between simulation and execution, such as current block time, number, hash etc.
In addition, a UserOperation is only allowed to access data related to this sender address: Multiple UserOperations should not access the same storage, so it is impossible to invalidate a large number of UserOperations with a single state change.

> There are 2 special contracts that interact with the account: the factory (initCode) that deploys the contract, and the paymaster that can pay for the gas.

Reviewer: ```suggestion
There are 2 special entity contracts that interact with the account: the factory (initCode) that deploys the contract, and the paymaster that can pay for the gas.
```

    bytes paymasterContext;
}

struct StakeInfo {
  uint256 stake;
  uint256 unstakeDelaySec;
}

```

> The `AggregatorStakeInfo` structure is further defined in [ERC-XXXX](link).

Reviewer 2: TODO add actual link

  * **code: -32503** - UserOperation out of time-range: either wallet or paymaster returned a time-range, and it has already expired (or will expire soon)
    * The `data` field SHOULD contain the `validUntil` and `validAfter` values
    * The `data` field SHOULD contain a `paymaster` value, if this error was triggered by the paymaster

>   * **code: -32504** - transaction rejected because paymaster is throttled/banned

Reviewer: irrelevant, as it is removed to an external ERC

Author: Removed all mentions of error codes.
